### ISSUE TITLE
Show the progressBar text over the player

### ISSUE THREAD

**Issue by 654wak654** [KoffeinFlummi/AGM#2182]
When a player has a progress bar for an action, for example things like "Bandaging Torso ...", "Repacking Magazines ...". The text could also show up above their head in 3D space. It'll make you aware what you're teammate is doing. So if he's bandaging the left leg of the wounded, you could straight go the left, if he's changing the left front tire, you could start changing the right front. Maybe another toggleable feature as a part of AGM_NameTags.

**Comment by jonpas** [comment]
I'd like this optional if it gets implemented, I believe communication is the way to go! :D

**Comment by 654wak654** [comment]
I can totally understand, I'd go with communication in some cases too. Already working on a little demo, it'll of course be a optional thing like the rest of the name tag features (toggleable in the AGM Options menu), don't worry.

**Comment by jonpas** [comment]
Yep exactly. I am thinking putting it into NameTags as an optional feature (by default disabled) with an option to enable inside the module (maybe even sub-options for what to show, eg. only process or add what's being worked on as well). That way when NameTags PBO is removed all NameTag related stuff is also gone (which I'd prefer personally as we don't use AGM Name Tags).

**Comment by 654wak654** [comment]
With the make-shift way that I'm doing it it's not possible to show more details than the action name. Someone who knows their way around agm could do that, but the only detail I could show is the progress which might be an overkill.

**Comment by jonpas** [comment]
Cool, AGM team will see how it works and give you feedback, they are great. ;)
